lib/services/android-debug-service.ts

```diff
@@ -174,7 +174,7 @@ class AndroidDebugService extends DebugServiceBase implements IPlatformDebugServ
 			debugerStarted = forwardsResult.indexOf(waitText) === -1;
 
 			if (!debugerStarted) {
-				sleep(500);
+				await sleep(500);
```

Comment:
I like the idea and I suggest to implement it in another PR (where we will add tests for it). We can use it on several places in CLI (at least two :) )
Thanks for the suggestion.